fdbt-dev/data/netexData/carnetReturn.xml

```diff
@@ -15,7 +15,7 @@
         <TypeOfFrameRef version="fxc:v1.0" ref="fxc:UK:DFT:TypeOfFrame_UK_PI_LINE_FARE_OFFER:FXCP"/>
         <NetworkFilterByValue>
           <objectReferences>
-            <OperatorRef version="1.0" ref="noc:BLAC"/>
+            <OperatorRef version="1.0" ref="noc:BLAC">noc:135742</OperatorRef>
```

Comment:
How come `noc:135742` is now necessary?

Reply:
OperatorRef is the noc code for the operator so this has correctly pulled through the NOC code whereas before it was incorrectly missing